src/Selenium2Driver.php

```diff
@@ -793,36 +787,23 @@ public function attachFile($xpath, $path)
     {
         $element = $this->findElement($xpath);
         $this->ensureInputType($element, $xpath, 'file', 'attach a file on');
-
-        // Upload the file to Selenium and use the remote path. This will
-        // ensure that Selenium always has access to the file, even if it runs
-        // as a remote instance.
-        try {
-          $remotePath = $this->uploadFile($path);
-        } catch (\Exception $e) {
-          // File could not be uploaded to remote instance. Use the local path.
-          $remotePath = $path;
-        }
-
-        $element->postValue(array('value' => array($remotePath)));
+        $element->sendKeys($path);
```

Comment:
Are you sure, that `sendKeys` is clever enough to handle file upload code on local/remote servers?